src/style.css

```diff
@@ -1 +1,114 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+:root {
+  --main-color: #3d4e61;
+}
+
+html {
+  font-family: Avenir, sans-serif;
```

Comment:
According to the checklist, you should remember to use fallback fonts - alternative font-family in case the main one doesn't work. You can add a fallback font like so: `font-family: Avenir, Arial, sans-serif;`